timur/docker-compose.yml

```diff
@@ -0,0 +1,62 @@
+version: "3.4"
+volumes:
+  app-gems:
+    driver: local
+  timur-db-data:
+    driver: local
+  timur-node-modules:
+    driver: local
+  timur-log-dir:
+    driver: local
+  timur-tmp-dir:
+    driver: local
+  timur-public-dir:
+    driver: local
+
+x-timur_base: &timur_base
+  build:
+    context: .
+    dockerfile: docker/app/Dockerfile.development
+  volumes:
+    - .:/app:cached
+    - app-gems:/app/vendor/bundle:rw
+    - ./node_modules:/app/node_modules:cached
+    - timur-log-dir:/app/log:rw
+    - timur-tmp-dir:/app/tmp:rw
+    - timur-public-dir:/app/public:rw
+    - ../etna:/etna:cached
+  environment: &app_environment
+    COVERAGE: "true"
+    DATABASE_HOST: "timur_db"
+    TIMUR_ENV: "development"
+
+services:
+  timur_app:
+    <<: *timur_base
+    ports:
+      - 3002:3000
```

Comment:
Change the host port we map to (3002), so it's different than the other containers.